client/blocks/reader-related-card-v2/style.scss

```diff
@@ -402,199 +398,18 @@ $reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
 	border-top: 1px solid lighten( $gray, 20% );
 	padding-top: 11px;
 
-	// Wrapper for site title, thumbnail, excerpt
+	// Wrapper for site title and excerpt
 	.reader-related-card-v2__post {
-		max-height: 282px;
+		max-height: 17px * 1.6 * 4;
 
 		&::after {
-			@include long-content-fade( $size: 30% );
+			@include long-content-fade( $size: 35px );
 			bottom: 0;
 			height: 22px;
 			top: auto;
 			visibility: visible;
 		}
 	}
-
-	&.is-same-site {
```

Comment:
@samouri it's a dupe of https://github.com/Automattic/wp-calypso/blob/b38cbab4381f5f2b0655efdac4fa48635c55d277/client/blocks/reader-related-card-v2/style.scss#L212. Basically lines 212 - 365 existed twice.